lib/mix/nerves/preflight.ex

```diff
@@ -38,12 +66,6 @@ defmodule Mix.Nerves.Preflight do
     end
   end
 
-  defp check_host_requirements(:darwin) do
```

Comment:
Was this accidentally left over from the move to preflight?

Reply:
It wasn't an accident, but yes it's being removed here because it's no longer needed since `check_platform!(:darwin)` (line 17) is already doing what this function used to do.